doc/installing-an-operating-system.md

```diff
@@ -18,3 +18,4 @@ Linux
 
  - [Fedora](distributions/fedora.md)
  - [NixOS](distributions/nixos.md)
+ * [Manjaro](distributions/manjaro.md)
```

Comment:
Let's keep this list alphabetically ordered. M goes before N :).

Yes, I could put NixOS front and center, but I'm trying to make this extremely distro-agnostic. Alphabetical order helps for that.